hera_qm/ant_metrics.py

```diff
@@ -10,6 +10,32 @@
 from hera_qm import utils
 
 
+ant_metrics_list = {'ant_metrics_meanVij': 'Mean of the absolute value of all '
```

Comment:
Yeah, I think putting them in a function is perfect. I also think having some broader documentation is helpful, but it would probably be too involved for a docstring. We should add the task of writing it to the issues page so it doesn't get lost in the shuffle.